External/Plugins/ProjectManager/PluginUI.cs

```diff
@@ -218,12 +215,40 @@ private void tree_BeforeLabelEdit(Object sender, NodeLabelEditEventArgs e)
         /// </summary>
         private void tree_AfterLabelEdit(Object sender, NodeLabelEditEventArgs e)
         {
+            string languageDisplayName = "(" + project.LanguageDisplayName + ")";
             if (!string.IsNullOrEmpty(e.Label) && Rename != null)
             {
-                if (!Rename((e.Node as GenericNode).BackingPath, e.Label))
+                if (e.Node is ProjectNode)
+                {
+                    var oldName = project.ProjectPath;
+                    string label = e.Label;
+                    int index = label.IndexOf(languageDisplayName);
+                    if (index != -1) label = label.Remove(index).Trim();
+                    string newName = string.Empty;
+                    try
+                    {
+                        newName = Path.Combine(project.Directory, label);
+                        newName = Path.ChangeExtension(newName, Path.GetExtension(oldName));
+                    }
+                    catch (Exception)
+                    {
+                        e.CancelEdit = true;
+                        isEditingLabel = false;
+                        return;
+                    }
+                    if (Rename(oldName, newName))
+                    {
+                        PluginBase.MainForm.OpenEditableDocument(newName);
+                        File.Delete(oldName);
```

Comment:
> Why do we have to delete the old file in fact?
Otherwise it turns out the two files, old and new

> Also this should be guarded by a try/catch.
today